interfaces/builtin/classic_support_test.go

```diff
@@ -81,8 +88,10 @@ func (s *ClassicSupportInterfaceSuite) TestUsedSecuritySystems(c *C) {
 	c.Check(string(snippet), testutil.Contains, "/usr/bin/systemd-run Uxr,\n")
 	c.Check(string(snippet), testutil.Contains, "/bin/systemctl Uxr,\n")
 	// connected plugs have a non-nil security snippet for seccomp
-	snippet, err = s.iface.ConnectedPlugSnippet(s.plug, s.slot, interfaces.SecuritySecComp)
+	seccompSpec := &seccomp.Specification{}
```

Comment:
This is also a common pattern it seems, I wonder if we could extract a test helper like `s.checkSeccompSnippet()` and a common base class? But probably work for a followup branch.